### PR TITLE
Can we remove undertow OIDC adapter?

### LINKED ISSUE #28788: Can we remove undertow OIDC adapter?
Pre-requisite is Wildfly OIDC adapter removed.

This may include testsuite module like `testsuite/integration-arquillian/servers/app-server/undertow`

### PR DESCRIPTION
**Pull request by douglaspalmer**
Closes #28788

### CODE REVIEW

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/authz/example/DefaultAuthzConfigAdapterTest.java [hunk 1]
```diff
@@ -48,13 +48,7 @@
 /**
  * @author <a href="mailto:psilva@redhat.com">Pedro Igor</a>
  */
-@AppServerContainer(ContainerConstants.APP_SERVER_WILDFLY)
-@AppServerContainer(ContainerConstants.APP_SERVER_EAP)
-@AppServerContainer(ContainerConstants.APP_SERVER_EAP6)
 @AppServerContainer(ContainerConstants.APP_SERVER_UNDERTOW)
```
Comment (jonkoops) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/authz/example/DefaultAuthzConfigAdapterTest.java:51]: So the idea is to remove support for all application servers aside from Undertow? That will eventually be replaced with Quarkus in the future?

**Review by jonkoops — DISMISSED** [review]

**Review by miquelsi — APPROVED** [review]

**Comment by mposolda** [comment]
The https://github.com/keycloak/keycloak/issues/29208 is pre-requisite of this one

**Comment by douglaspalmer** [comment]
> The #29208 is pre-requisite of this one

@mposolda I rebased this and it no longer effect Policy Enforcer tests. I think we can merge this. Policy Enforcer tests will be effected by removing keycloak-adapter-core. So #29189 will need to be completed before removing keycloak-adapter-core.

**Review by jonkoops — APPROVED** [review]

**Review by mposolda — APPROVED** [review]
@douglaspalmer @jonkoops @miquelsi Thanks!

**Pull request merged**